README.md

```diff
@@ -39,6 +39,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | primary_metastore_whitelist                  | List of Hive databases to whitelist on primary Metastore.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |  list  |                `<list>`                |    no    |
 | primary_metastore_mapped_databases           | List of Hive databases mapped from primary Metastore.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |  list  |                `<list>`                |    no    |
+| primary_metastore_host_read_only             | Optional Primary Hive Metastore READ ONLY hostname configured in Waggle Dance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | string |                  ``                    |    no    |
```

Comment:
`primary_metastore_read_only_host` and `primary_metastore_read_only_port` reads better.

Reply:
done